_meta/beat.yml

```diff
@@ -10,6 +10,7 @@ apm-server:
   #max_header_bytes:
   #read_timeout: 2s
   #write_timeout: 2s
+  #shutdown_timeout: 5s
```

Comment:
Reminds me of: https://github.com/elastic/beats/blob/master/filebeat/_meta/common.reference.p2.yml#L254 Nice that we have the same naming.